ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -107,6 +106,10 @@ def initialize(opts = {})
       @oidc_settings                    = opts.fetch(:oidc_settings, {})
     end
 
+    def servername
+      @servername || OodPortalGenerator.fqdn
+    end
```

Comment:
On Linux I can't think of a situation where hostname be not available and thus `Socket.gethostbyname` would fail.  Even when one is not defined, you get `localhost`.

Reply:
Yep looks like a quick google search only turned up stuff for Mac.